.github/workflows/docker.yml

```diff
@@ -0,0 +1,32 @@
+name: On push to main add a tag to the commit if docker or renv stuff changed
+
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Tag in case renv or docker files changed
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # OR "2" -> To retrieve the preceding commit.
+
+      - name: Get changed files
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v35
+        with:
+          files: |
+            dockerfile
+            docker-compose.test.yml
```

Comment:
Why should a new docker be build after changes in `docker-compose.test.yml`?  This is just a local test to test in this repo if a docker image can be build, so I see no need to create a tag to build a new docker in this case.

Reply:
You're right. I fixed it.